model_analyzer/record/types/gpu_power_usage.py

```diff
@@ -85,7 +85,7 @@ def __lt__(self, other):
         the other
         """
 
-        return self.value() < other.value()
+        return other.value() < self.value()
```

Comment:
It feels like these should all be in the root class so there is a single point of failure. I won't make you do that this story, but it is something that we should consider in the future

Reply:
I thought about that, but it seemed beyond the scope of this story